byoeb-v1/byoeb/byoeb/apis/admin.py

```
@@ -57,20 +69,20 @@ async def stream_csv() -> AsyncIterator[bytes]:
     })
 
 @admin_apis_router.post("/experiment")
-async def query_handler(input: QueryInput) -> JSONResponse:
+async def experiment_form(input: QueryInput) -> JSONResponse:
```

Ruff (0.15.11)

[error] Function argument `input` is shadowing a Python builtin

(A002)